common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/MqttTransportHandler.java

```diff
@@ -1082,10 +1085,12 @@ private void checkGatewaySession(SessionMetaData sessionMetaData) {
             if (infoNode != null) {
                 JsonNode gatewayNode = infoNode.get("gateway");
```

Comment:
it is better to use a constant instead of a raw value because there are many `"gateway"` usages across the codebase